README.md

```diff
@@ -16,30 +16,33 @@
 
 ### General
 
-#### Contact Info:
+#### Quick Links:
 
-Email List: sf.mentors@devbootcamp.com
+- [End of Session Form](https://docs.google.com/a/devbootcamp.com/forms/d/1bNIBtgy2ephY5117eHa31iFVgVRxPJAA0zzyeEqvTlA/viewform)
 
-Slack: sfmentors
+- [Student Notes](https://mentor.devbootcamp.com/)
```

Comment:
Different than link above with same name. Not saying it will cause confusion, but...